app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -47,6 +47,18 @@ def index
 
     private
 
+    def pending_profile_policy
+      @pending_profile_policy ||= PendingProfilePolicy.new(
+        user: current_user,
+        resolved_authn_context_result: resolved_authn_context_result,
+        biometric_comparison_requested: biometric_comparison_requested?,
+      )
+    end
+
+    def user_has_useable_pending_profile?
```

Comment:
TIL "useable" can be considered a valid spelling, but from this resource it seems without the "e" is more common in American English: https://capitalizemytitle.com/usable-vs-useable/

```suggestion
    def user_has_usable_pending_profile?
```